app/assets/stylesheets/page_specific/reports.css

```diff
@@ -0,0 +1,25 @@
+/* reports.css */
+.report-year {
+		display: block;
+		text-align: center;
+}
+
+.monthly.col-md-4.col-sm-6 {
+		padding-left: 0px !important;
+		padding-right: 0px !important;
+		padding-top: 20px;
+		padding-bottom: 20px;
+	}
+
+.report-preview:hover {
+		transform: scale(1.1);
+}
+
+h {
+	float: left;
+	display: block;
+}
+
+h4 {
+	font-size: 22px !important;
```

Comment:
This will override the entire site's h4 styling. Please use an `id` instead of the `h4` tag.